ci/test_wheel_dask_cudf.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+set -eoxu pipefail
```

Comment:
```suggestion
set -euo pipefail
```

Do we need the `-x` here? Or can we keep it consistent with the other scripts?